stubs/docutils/docutils/nodes.pyi

```diff
@@ -34,6 +94,17 @@ class Element(Node):
     def __iadd__(self: Self, other: Node | Iterable[Node]) -> Self: ...
     def __getattr__(self, __name: str) -> Any: ...  # incomplete
 
+class Text(Node, str):
+    tagname: str
+    children: tuple[()]
```

Comment:
Both of these should probably marked as `ClassVar`.

Reply:
I added it for `tagname`. For `children` I cannot because then mypy.stubtest fails with:

> Cannot override instance variable (previously declared on base class "Node") with class variable